lib/rolify/adapters/active_record/resource_adapter.rb

```diff
@@ -26,7 +26,7 @@ def resources_find(roles_table, relation, role_name)
       end
 
       def in(relation, user, role_names)
-        roles = user.roles.where(:name => role_names).select("#{quote_table(role_class.table_name)}.#{quote_column(role_class.primary_key)}").to_a
+        roles = user.roles.where(:name => role_names).select("#{quote_table(role_class.table_name)}.#{quote_column(role_class.primary_key)}")
```

Comment:
What does it provide? is it using a subquery instead of flattening the array in the query?

Reply:
Only will fix the problem with Oracle what limit the maximum number of expressions in a list what is 1000.